presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -652,6 +653,30 @@ private Type getVarcharType(Expression value, StackableAstVisitorContext<Context
         @Override
         protected Type visitSubscriptExpression(SubscriptExpression node, StackableAstVisitorContext<Context> context)
         {
+            Type baseType = process(node.getBase(), context);
+            // Subscript on Row hasn't got a dedicated operator. Its Type is resolved by hand.
+            if (baseType instanceof RowType) {
+                if (!(node.getIndex() instanceof LongLiteral)) {
+                    throw new SemanticException(INVALID_PARAMETER_USAGE, node.getIndex(),
```

Comment:
nit: Either merge the next line, or keep on parameter a separate line, like this:

```
new SemanticException(
        INVALID_PARAMETER_USAGE, 
        node.getIndex(),
         "Subscript expression on ROW requires a constant index");
```

ditto for other exception construction.

Reply:
got it, I had the intellij Airlift codestyle but looks like I have to edit a property "Keep Line breaks during formatting" for it to autoformat.